modules/object-cache/health/load.php

```diff
@@ -0,0 +1,225 @@
+		),
+		'actions'     => sprintf(
+			'<p><a href="%s" target="_blank" rel="noopener">%s <span class="screen-reader-text">%s</span><span aria-hidden="true" class="dashicons dashicons-external"></span></a></p>',
+			esc_url( $action_url ),
+			__( 'Learn more about persistent object caching.', 'performance-lab' ),
+			/* translators: Accessibility text. */
+			__( '(opens in a new tab)', 'performance-lab' )
+		),
+	);
+
+	if ( wp_using_ext_object_cache() ) {
+		$result['label'] = __( 'A persistent object cache is being used', 'performance-lab' );
+
+		return $result;
+	}
+
+	/**
+	 * Filter whether to suggest using a persistent object cache.
+	 *
+	 * Plugin and theme authors should NOT use this filter to discourage the use of an object cache.
+	 *
+	 * @param bool $suggest
+	 */
+	if ( ! apply_filters( 'site_status_suggest_persistent_object_cache', false ) ) {
+		$result['label'] = __( 'A persistent object cache is not required', 'performance-lab' );
+
+		return $result;
+	}
+
+	$available_services = oc_health_available_object_cache_services();
+
+	$notes = __( 'Speak to your web host about what persistent object caches are available and how to enable them.', 'performance-lab' );
+
+	if ( ! empty( $available_services ) ) {
+		$notes .= sprintf(
+			/* translators: Available object caching services. */
+			__( ' Your host appears to support the following object caching services: %s.', 'performance-lab' ),
+			implode( ', ', $available_services )
+		);
+	}
+
+	/**
+	 * Filter the second paragraph of the health check's description
+	 * when suggesting the use of a persistent object cache.
+	 *
+	 * Hosts may want to replace the notes to recommend their preferred object caching solution.
+	 *
+	 * Plugin authors may want to append notes (not replace) on why object caching is recommended for their plugin.
+	 *
+	 * @param string $description
+	 * @param array $available_services
+	 */
+	$notes = apply_filter( 'site_status_persistent_object_cache_notes', $notes, $available_services );
+
+	$result['status']       = 'recommended';
+	$result['label']        = __( 'You should use a persistent object cache', 'performance-lab' );
+	$result['description'] .= sprintf(
+		'<p>%s</p>',
+		wp_kses(
+			$notes,
+			array(
+				'a'      => array( 'href' => true ),
+				'code'   => true,
+				'em'     => true,
+				'strong' => true,
+			)
+		)
+	);
+
+	return $result;
+}
+
+/**
+ * Callback for `site_status_persistent_object_cache` filter.
+ *
+ * Determines whether to suggest using a persistent object cache.
+ *
+ * @param mixed $should_suggest Whether to suggest using a persistent object cache.
+ * @return bool
+ */
+function oc_health_should_persistent_object_cache( $should_suggest ) {
+	global $wpdb;
+
+	$thresholds = array(
+		'alloptions_count' => 500,
+		'alloptions_bytes' => 100000,
```

Comment:
Does this step on the toes of #92 CC @manuelRod @pbearne

Reply:
In a sense yes, but #92 only warns the user about it.

This module explicitly recommends and object cache to the user.